src/Tables/Table.tsx

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 import { ReactElement, ForwardRefExoticComponent } from 'react';
 import TableHead from './TableHead';
 import TableBody from './TableBody';
+import { fromTheme } from '../Theme';
```

Comment:
Last little thing: This import of `fromTheme` should be removed if it's not being used.